### PR TITLE
Refactor condition of CPU sched

### PR DESCRIPTION
**Pull request by ventureoo**
Notice:
- hardened is renamed to bore-hardened for clarity (well, there is no such scheduler as "hardened")
- Introduced a local `_die()` function to exit the build

### CODE REVIEW

**Review by ptr1337 — APPROVED** [review]
LGTM

**Review by sirlucjan — APPROVED** [review]
LGTM

File: linux-bore/PKGBUILD [hunk 1]
```diff
@@ -248,39 +248,22 @@ if [ -n "$_latency_nice" ]; then
...
+    pds|bmq) # BMQ/PDS scheduler
+        source+=("${_patchsource}/sched/0001-prjc-cachy.patch");;
+    tt) ## TT Scheduler
+        source+=("${_patchsource}/sched/0001-tt-cachy.patch");;
+    cacule|cacule-rdb) ## CacULE(-RDB) Scheduler
+        source+=("${_patchsource}/sched/0001-cacULE-cachy.patch");;
+    bore) ## BORE Scheduler with latency_nice
+        [ -n "$_tune_bore" ] && source+=("${_patchsource}/misc/0001-bore-tuning-sysctl.patch")
+        source+=("${_patchsource}/sched/0001-bore-cachy.patch");;
+    bore-hardened) ## Hardened Patches with BORE Scheduler
```
Comment (ventureoo) [inline linux-bore/PKGBUILD:262]: In original condition, the bore-tuning-sysctl patch was also not applying for hardened. Is that normal?

Comment (ptr1337) [reply linux-bore/PKGBUILD:262]: I need to check if it is applying well, otherwise we can add it

Comment (ventureoo) [reply linux-bore/PKGBUILD:262]: OK.

**Pull request merged**